style(*): grammar, spelling, punctuation (#539)
discordjs/guide: guide/commando/README.md

	.registerCommandsIn(path.join(__dirname, 'commands'));
```

> Doing this, you've also created your first command group! Make another folder called `first` in your `commands` folder so that you can put the commands for that group in there. The group will be displayed in the help command as `Your First Command Group`. You can use any name you want for either of these options, but do note that the key (`first`) **must be lowercase**!

Reviewer: ```suggestion
You've just created your first command group! Make another folder called `first` in your `commands` folder so that you can put the commands for that group in there. The group will be displayed in the help command as `Your First Command Group`. You can use any name you want for either of these options, but do note that the key (`first`) **must be lowercase**!
```
